Client/qtTeamTalk/mytableview.cpp

```diff
@@ -17,9 +17,14 @@
 
 #include "mytableview.h"
 #include <QAccessible>
+#include <QShortcut>
 
 MyTableView::MyTableView(QWidget* parent/* = nullptr*/) : QTableView(parent)
 {
+    QShortcut* shortcutLeft = new QShortcut(QKeySequence(Qt::CTRL | Qt::Key_Left), this);
```

Comment:
Shouldn't this be documented somehow? Or is it a generally know shortcut?

Reply:
Yes, it could be better to document this, but where can we do it?

Reply:
In https://github.com/BearWare/TeamTalk5/blob/master/Documentation/TeamTalk/userinterface.md section "User Accounts dialog".

Reply:
OK, done! We should also update doc with new dialogs, but probably in a future PR or commit